tests/Integration/Command/GetOfficesTest.php

```diff
@@ -0,0 +1,131 @@
+
+        $response = $command->getOffices(new GetOfficesRequest());
+
+        $this->assertCount(1, $response->getOffices());
+        $this->assertOffice($response);
+    }
+
+    /**
+     * @test
+     */
+    public function responseWithError(): void
+    {
+        $this->expectException(MalformedResponseException::class);
+        $this->expectExceptionMessage('Error response');
+
+        $command = $this->getCommand();
+        $this->mockGuzzleResponse(new Response(200, [], $this->getErrorBody()));
+
+        $response = $command->getOffices(new GetOfficesRequest('', 90));
+
+        $this->assertCount(0, $response->getOffices());
+        $this->assertOffice($response);
+    }
+
+    private function assertOffice(GetOfficesResponse $response): void
+    {
+        $office = $response->getOffices()->get(0);
+
+        $this->assertNotNull($office);
+        $this->assertSame($office->id, 33701);
+        $this->assertSame($office->code, '15774');
+        $this->assertSame($office->isMPS, false);
+        $this->assertSame($office->isAPS, false);
+        $this->assertSame($office->name, 'ACS AFTHIMERON ATTIKIS');
+        $this->assertSame($office->nameEn, 'ACS AFTHIMERON ATTIKIS');
+        $this->assertSame($office->phones, ['2107714680']);
+        $this->assertSame($office->emails, ['shops@acscourier.gr']);
+        $this->assertSame($office->address->id, null);
+        $this->assertSame($office->address->city->id, 55978);
+        $this->assertSame($office->address->city->country->id, 1084);
+        $this->assertSame($office->address->city->country->code2, 'GR');
+        $this->assertSame($office->address->city->country->code3, 'GRC');
+        $this->assertSame($office->address->city->country->name, 'Гърция');
+        $this->assertSame($office->address->city->country->nameEn, 'Greece');
+        $this->assertTrue($office->address->city->country->isEU);
+        $this->assertSame($office->address->city->postalCode, '10431');
+        $this->assertSame($office->address->city->name, 'Αθήνα');
+        $this->assertSame($office->address->city->nameEn, 'ATHINA');
+        $this->assertSame($office->address->city->phoneCode, null);
+        $this->assertSame($office->address->city->expressDeliveries, null);
+        $this->assertSame($office->address->fullAddress, ' Αθήνα Bairaktari 15');
+        $this->assertSame($office->address->num, '');
+        $this->assertSame($office->address->other, 'Bairaktari 15');
+        $this->assertSame($office->address->latitude, 37.983696666667);
+        $this->assertSame($office->address->longitude, 23.771268333333);
+        $this->assertSame($office->info, '');
+        $this->assertSame($office->currency, 'EUR');
+        $this->assertNull($office->language);
+        $this->assertEquals($office->openingHours, new OpeningHours(1627534800000, 1627578000000));
+        $this->assertEquals($office->halfDayOpeningHours, new OpeningHours(1627538400000, 1627552800000));
```

Comment:
There's no test for `OpeningHours`, I'd definitely add one given there's a transformation happening. Or maybe assert datetimes here